pipelines.md

```diff
@@ -206,9 +206,9 @@ Different environments support slightly different variants of Markdown - Remarka
 
 Markdown has taken over the world of technical writing. Using the same format everywhere creates tremendous opportunities, so I highly recommend that you write your documentation in Markdown. With the same text, you can generate:
 
-- _Digital notes_. [Notion](https://notion.so/), [notable](https://notable.app/), GitHub.
+- _Digital notes_. [Notion](https://notion.so/), [notable](https://notable.app/), [Obsidian](https://obsidian.md/), [HackMD](https://hackmd.io/), GitHub.
 - _Blogs_. [Wordpress](https://wordpress.com), [Jekyll](https://jekyllrb.com/)
-- _Wikis_. Github.
+- _Wikis_. GitHub.
 - _Static sites_. [Jekyll](https://jekyllrb.com/), [eleventy](https://11ty.dev/), GitHub Pages
 - _Executable books_. [jupyterbook](https://jupyterbook.org/) generates this book.
 - _Slide decks_. [Pandoc](https://pandoc.org/) via Beamer.
```

Comment:
looking at it now, it might not be obvious that Beamer references the LaTeX package - would it be worth adding a hyperlink for that as well? Though there isn't a good homepage as far as I can see, just the CTAN package. how about
```suggestion
- _Slide decks_. [Pandoc](https://pandoc.org/) via conversion to LaTeX/Beamer.
```